CMakeLists.txt

```diff
@@ -76,6 +76,13 @@ if(CMAKE_SIZEOF_VOID_P EQUAL 8 AND NOT WIN32)
    endif()
 endif()
 
+if(CMAKE_SIZEOF_VOID_P EQUAL 8 AND NOT WIN32)
```

Comment:
these don't seem to gate much of the compile. For example when doing a win32 build I get

```
/home/spoon/eos.win32/libraries/eos-vm/include/eosio/vm/allocator.hpp:11:10: fatal error: sys/mman.h: No such file or directory
 #include <sys/mman.h>
          ^~~~~~~~~~~~
```